bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDirsMonitor.java

```diff
@@ -229,5 +231,18 @@ private void checkDirs(final LedgerDirsManager ldm)
         }
         ldm.getWritableLedgerDirs();
     }
+
+    @VisibleForTesting
+    void setDiskSpaceThreshold(float diskSpaceThreshold, float diskUsageLwmThreshold) {
```

Comment:
do we really need this method? can we instantiate a LedgerDirsMonitor to run tests?

Reply:
Make sense to me. I have addressed your comments. PTAL.